src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx

```diff
@@ -27,9 +28,9 @@ export const DeploySuccess = ({ projectId }: { projectId: number }) => {
    */
   const twitterShareUrl = useMemo(() => {
     let juiceboxUrl = `https://juicebox.money/v2/p/${projectId}`
-    const chainId = chain?.name.toLowerCase() ?? 'mainnet'
-    if (chainId !== 'mainnet') {
-      juiceboxUrl = `https://${chainId}.juicebox.money/v2/p/${projectId}`
+    const chainId = parseInt(chain?.id ?? MAINNET_CHAIN_ID.toString())
+    if (chainId !== MAINNET_CHAIN_ID) {
+      juiceboxUrl = `https://${NETWORKS[chainId].name}.juicebox.money/v2/p/${projectId}`
```

Comment:
**suggestion** I think using our `readNetwork` constant is safer and less complicated here.